academic-services/project-academic-knowledge/reference-interpret-method.md

```diff
@@ -2,7 +2,7 @@
 title: Interpret method
 description: Generates semantic interpretations of a natural language query
 ms.topic: reference
-ms.date: 2020-03-19
+ms.date: 2020-03-25
```

Comment:
>2020-03-25 [](start = 9, length = 10)

FYI - Have you seen the build warnings for our date format?  I think they want them in mm/dd/yyyy format now...

Reply:
How very American of them.

---
In reply to: [398027232](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/135#discussion_r398027232) [](ancestors = 398027232)